config.json

```diff
@@ -118,7 +118,7 @@
     },
     "build-managed":{
       "alias":{
-        "s":{
+        "binaries":{
```

Comment:
Nevermind I think what you have is good for now. I now see the default empty settings are under "defaultValues", and changing those to this would be a break so at least for now we should keep what you have.